src/docs/reference/modules/ContentFields/README.md

```diff
@@ -1,33 +1,46 @@
 # Content Fields (`OrchardCore.ContentFields`)
 
-This module provides common content fields.  
-Some fields are available in their specific module.
+This module provides common content fields for capturing data in Orchard Core. Fields are typically added to a content part, which is then added to a content type. 
+
+!!! note
+    While the UI allows adding fields directly to content types, behind the scenes it creates a content part with the same name as the content type and attaches the fields to it.
+
+Fields support multiple editors - different UIs for data input. For example, a TextField can have editors like `Email`, `TextArea`, or `Tel`. Editors can be configured in the field settings.
 
 ## Available Fields
 
-| Name                                | Properties                                                     |
-|-------------------------------------|----------------------------------------------------------------|
-| `BooleanField`                      | `bool Value`                                                   |
-| `ContentPickerField`                | `string[] ContentItemIds`                                      |
-| `DateField`                         | `DateTime? Value`                                              |
-| `DateTimeField`                     | `DateTime? Value`                                              |
-| `HtmlField`                         | `string Html`                                                  |
-| `LinkField`                         | `string Url, string Text`                                      |
-| `LocalizationSetContentPickerField` | `string[] LocalizationSets`                                    |
-| `MarkdownField`                     | `string Markdown`                                              |
-| `MediaField`                        | `string[] Paths`                                               |
-| `MultiTextField`                    | `string[] Values`                                              |
-| `NumericField`                      | `decimal? Value`                                               |
-| `GeoPointField`                     | `decimal Latitude, decimal Longitude`                          |
-| `TaxonomyField`                     | `string TaxonomyContentItemId, string[] TaxonomyContentItemId` |
-| `TextField`                         | `string Text`                                                  |
-| `TimeField`                         | `TimeSpan? Value`                                              |
-| `UserPickerField`                   | `string[] UserIds`                                             |
-| `YoutubeField`                      | `string EmbeddedAddress, string RawAddress`                    |
+| Name                                | Properties                                                     | Available Editors |
```

Comment:
The formatting is off here. I guess all of the 

```
Standard ``
```

were meant to be

```
`Standard`
```
?

Reply:
The default editor is named "Standard" and the value for the Standard is empty string. So (Standard ``) shows the name and the corresponding value for each. So I think it's correct

Reply:
Then something like "Standard (empty)" would be clearer anyway. But the main problem here is that due to the backticks the code formatting is broken (check out by running the docs site locally).

Reply:
@Piedone done.